lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4705,13 +4705,14 @@ DefMacroI('\nfss@catcodes', undef, <<'EOMacro');
    \@makeother\'%
    \@makeother\"%
 EOMacro
-
+DefMacroI('\ltx@hard@MessageBreak',undef, '^^J');
 sub make_message {
   my ($cmd, @args) = @_;
   my $stomach = $STATE->getStomach;
   $stomach->bgroup;
   Let('\protect', '\string');
-  my $message = join("\n", map { ToString(Expand($_)) } @args);
+  Let('\MessageBreak','\ltx@hard@MessageBreak'); # tricky, we need Expand() to execute it
+  my $message = join("", map { ToString(Expand($_,T_CS('\MessageBreak'))) } @args);
```

Comment:
This is a bit of a perverse change. The warning messages of etoolbox have a trailing `\@gobble` that seems to intend to remove the trailing line break. So to avoid an argument error, I had to add the trailing newlines in. Meanwhile I also noticed `\MessageBreak` had to be locally bound to `^^J` (and expandable), but wasn't, so fixed that too.